src/Simplex/Chat.hs

```diff
@@ -276,9 +276,14 @@ agentSubscriber = do
 subscribeUserConnections :: (MonadUnliftIO m, MonadReader ChatController m) => m ()
 subscribeUserConnections = void . runExceptT $ do
   user <- readTVarIO =<< asks currentUser
+  subscribeNonActivatedConnections user
   subscribeContacts user
   subscribeGroups user
   where
+    subscribeNonActivatedConnections user = do
+      connections <- withStore (`getNullContactConnections` user)
+      forM_ connections $ \conn ->
```

Comment:
You can also have `Connection {agentConnId}`

Reply:
and you need `catchError` as below, even if the handler does nothing (I don't think there is a meaningful error message here)